q0_gui.py

```diff
@@ -204,9 +204,11 @@ def handle_cal_error(self, message):
     def load_calibration(self):
         if self.selectedCM.name not in self.cal_option_windows:
             option_window: Display = Display()
+            option_window.setWindowTitle(f"CM {self.selectedCM.name} Calibration Options")
             cal_options = q0_gui_utils.CalibrationOptions(self.selectedCM)
             cal_options.cal_loaded_signal.connect(self.handle_cal_status)
             cal_options.cal_loaded_signal.connect(partial(self.ui.rf_groupbox.setEnabled, True))
+            cal_options.cal_loaded_signal.connect(partial(self.ui.show_cal_data_button.setEnabled, True))
```

Comment:
so this will enable the button when an old calibration is loaded. Is the button also enabled when I take a new calibration? I don't see that anywhere. Am I missing something?

Reply:
Oops